reference/tls/cleartext-redirection.md

```diff
@@ -21,7 +21,38 @@ Ambassador Edge Stack exposes configuration for this in two ways:
 
 Typically, port-based redirection is the preferred method since it is simpler to manage and will work with all use cases. Redirecting based off of the `x-forwarded-proto` header requires an L7 load-balancer or proxy in front of Ambassador Edge Stack to set that header.
 
-## Port-based Redirection
+## Using the `Host` resource
+
+As of Ambassador API Gateway and Edge Stack 1.0.0, the [`Host`](/reference/host-crd) resource is used to configure how Ambassador handles cleartext on a domain. You can now configure Ambassador to deny, allow, or redirect cleartext to HTTPS on a domain using a single resource.
+
+The following `Host` gives an example of how to enable Ambassador to redirect cleartext to HTTPS on the `host.example.com` domain. 
+
+```yaml
+---
+apiVersion: getambassador.io/v2
+kind: Host
+metadata:
+  name: example-host
+spec:
+  hostname: host.example.com
+  acmeProvider:
+    email: julian@example.com
+  insecure:
+    action: Redirect
+```
+
+**Note:** The default behavior when Ambassador is terminating TLS is to redirect cleartext for all `Host`s
+
+## For Ambassador 0.86.1 and below
+
+In Amabassador 0.86.1 and below, TLS behavior was configured with the `TLSContext`. This included cleartext redirection. 
+
+The follow reference is **deprecated** and for people using old versions of Ambassador.
```

Comment:
Using `TLSContext` to do so is now deprecated. For those using older versions of Ambassador, [upgradeto the latest verion](/user-guide/upgrade-to-edge-stack/) of Ambassador to use the `Host` resource to manage TLS termination.